code/game/objects/effects/decals/Cleanable/misc.dm

```diff
@@ -201,6 +201,14 @@
 	icon = 'icons/effects/tomatodecal.dmi'
 	random_icon_states = list("smashed_pie")
 
+/obj/effect/decal/cleanable/spaghetti_spill
+	name = "spilled spaghetti"
+	desc = "A result of bursting into treats from awkward situations."
+	density = 0
+	anchored = 1
+	icon = 'icons/effects/tomatodecal.dmi'
```

Comment:
maybe you should make `reagent = FLOUR` or `TOMATOJUICE`

Reply:
Oh okay

Reply:
Done